hbase-server/src/main/java/org/apache/hadoop/hbase/master/balancer/StochasticLoadBalancer.java

```diff
@@ -263,6 +263,10 @@ protected void setCandidateGenerators(List<CandidateGenerator> customCandidateGe
     this.candidateGenerators = customCandidateGenerators;
   }
 
+  public List<CandidateGenerator> getCandidateGenerators() {
```

Comment:
Since we are allowing mutating the list, can you add a comment that this method is only for test purpose?

Reply:
Yes, I'll change it.